### DISCUSSION TITLE
Is System Informer able to show what is causing delays with opening, saving, renaming files and folders?

### DISCUSSION THREAD

**Discussion by Almighty188** [winsiderss/systeminformer#2181] [Questions / Answers]
Is System Informer able to show what is causing delays with opening, saving, renaming files and folders as I seem to have a 5-10 second and sometimes up to a minute pause when I am opening, saving, renaming files and folders.  If it can be done, how would I do it?

**Comment by dmex** [comment]
It could be a number of issues:
* BIOS configuration (XMP or AMP should be enabled).
* Incorrect, outdated or missing motherboard drivers and/or old firmware (AMD has known issues with fTPM and requires firmware/driver update)
* Device security - Most processors can't handle the Spectre 'fixes' and should be disabled for home users.
* Also check System Informer > CPU > Latency is below 5 and consider switching to type-2 virtualization.
* File system filters - Check fltmc and fltmc instances.
* Legacy interrupts - Device Manager > View > Devices by connection > Expand Interrupt request (IRQ) > Devices starting with (PCI) and ending with a positive number (1-100) have reverted to legacy mode.
* Excluding all the above check processes for I/O activity and handle activity  and/or check autoruns for shell extensions.
